src/sql/azdata.d.ts

```diff
@@ -5,6 +5,7 @@
 
 declare module 'azdata' {
 	import * as vscode from 'vscode';
+	import { OperatingSystem } from 'vs/base/common/platform';
```

Comment:
We can't import from our source files here, this will break extensions trying to use it. only import from vscode is allowed since we expect people to have the definition files for vscode as well as ours.

Reply:
The only alternative here is to just use "integer" instead of the int-based enum then. Would you recommend that?

Reply:
Changed data type from OperatingSystem enum to number. It works the same, and does not require the import. Import is removed.